HelloWorld.scala

```diff
@@ -0,0 +1,12 @@
+object HelloWorld {
+   /* This is my first java program.  
+    * This will print 'Hello World' as the output
+    * This is an example of multi-line comments.
+    */
+   def main(args: Array[String]) {
+      // Prints Hello World
+      // This is also an example of single line comment.
+      var name = "sina";
```

Comment:
change to a name of someone who is not a scrub, like Steven Sheffey

Reply:
hmmm :thonking: